app/controllers/Application.scala

```diff
@@ -598,7 +598,7 @@ object Application extends Controller {
   def appNameToDisplay(metadata: Option[Metadata], notebookPath: Option[String]): String = {
     val explicitName = metadata.map(_.name).getOrElse("Spark-notebook")
     notebookPath match {
-      case Some(path) => s"${explicitName} ($path)"
+      case Some(path) => s"Notebook: $path"
```

Comment:
just `folder/something.snb` does not make it clear it's a notebook.
I'd vote for `Spark notebook: $path`, but rather long.